Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "TreeParzen"
 uuid = "eb66a70c-a255-11e9-03ea-7ba6b2f22006"
 authors = ["iqml <https://github.com/iqml>"]
```

Comment:
Wanna update author while you're here?

Reply:
Actually my bad, I thought the link wasn't valid but it is

Reply:
I think it's fine for now. The link is valid for the `iqml` and we might just extend the list at some point  :)